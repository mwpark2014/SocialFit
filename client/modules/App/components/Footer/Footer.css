.footer{
  text-align: center;
  padding: 12px 0;
  background-color: #1F0A00;
}

.footer p{
  margin: 0 0 8px 0;
  font-size: 14px;
  color: #ffffff;
  font-family: Georgia,Serif;
}

.footer a{
  text-decoration: none;
  font-weight: 700;
}
