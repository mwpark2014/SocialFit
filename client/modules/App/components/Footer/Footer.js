import React from 'react';

// Import Style
import styles from './Footer.css';


export function Footer() {
  return (
    <div className={styles.footer}>
      <p>&copy; 2018 &middot; Mason Park</p>
    </div>
  );
}

export default Footer;
