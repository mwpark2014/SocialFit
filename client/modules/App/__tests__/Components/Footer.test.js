import React from 'react';
import Footer from '../../components/Footer/Footer';
import renderer from 'react-test-renderer';

test('Footer snapshot', () => {
  const component = renderer.create(
    <Footer />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot(); // eslint-disable-line
});
