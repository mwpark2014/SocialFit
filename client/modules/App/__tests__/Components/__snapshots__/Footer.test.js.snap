// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot 1`] = `
<div
  className={undefined}
>
  <p>
    © 2018 · Mason Park
  </p>
</div>
`;
